### DISCUSSION TITLE
Starred topics != Mastered & Known

### DISCUSSION THREAD

**Discussion by abrahamnm** [lowlighter/metrics#636] [General]
While setting up my yaml file I realized that the Topics plugin maybe doesn't have the most appropriate title when you want to display icons instead of labels.

I have starred many projects that interest me in different technologies and languages, but that doesn't necessarily mean that I am a master and know them.

English is not my native language, so I don't know if it's the best choice, but "Technologies and topics of interest" might be an option.

**Comment by lowlighter** [comment]
Yeah, we should probably just add a mode `icons` which has the same effect as `mastered` except the title would change

**Reply by lowlighter** [reply]
This has been implemented through #648
